go-pkg/ds/permission_test.go

```diff
@@ -0,0 +1,47 @@
+package ds
+
+import (
+	"log"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+)
+
+func init() {
+	log.SetFlags(log.Llongfile)
+}
+
+func createUser() *User {
```

Comment:
Maybe `createTestUser`. This would help avoid someone using this function in non-testing code.